### DISCUSSION TITLE
Can I disable this specific warning?

### DISCUSSION THREAD

**Discussion by fernandolucchesi** [vercel/next.js#31340] [Help] [answered]
I find this to be very annoying because I am using an `ImageWrapper` component with `position: absolute`, where its parent has the `position: relative` property instead. It is working perfectly but my console is now full of warnings.

https://github.com/vercel/next.js/blob/cf206a8392dd84bae09d1be0237c954a563cff93/packages/next/client/image.tsx#L292

**Comment by Anonymous-dev-v** [comment]
Totally agreee, have tried everything to remove that warning, nothing works !

**Comment by justlevine** [comment]
I cant believe there isnt a solution for this yet

**Comment by dertuman** [comment]
I too need a solution for this! Not happy with the Image component overall, regarding styling.

**Comment by styfle** [comment, marked as answer]
Thanks for the feedback!

We just released a [couple](https://github.com/vercel/next.js/pull/34551) [fixes](https://github.com/vercel/next.js/pull/34562) on the canary channel in [v12.1.1-canary.1](https://github.com/vercel/next.js/releases/tag/v12.1.1-canary.1).

You can try it out today with `npm install next@canary` or `yarn add next@canary`.
